src/pages/Channel/List.js

```diff
@@ -1,8 +1,17 @@
 import React from "react";
+import { useHistory } from "react-router-dom";
 import PropTypes from "prop-types";
 
 export default function List({ text, id }) {
-  return <li id={id}>{text}</li>;
+  const history = useHistory();
+  const handelClick = () => {
+    history.push(`/lobby/${id}`);
+  };
+  return (
+    <li id={id} onClick={handelClick} onKeyDown={handelClick} aria-hidden="true">
```

Comment:
혹시 하나의 함수로 둘다 사용하는 것이라면 함수명에 대해 고민해보면 좋을 것 같습니다. 
```suggestion
    <li id={id} onClick={handelClick} onKeyDown={handleKeyDown} aria-hidden="true">
```